gitcoins-cordapp/build.gradle

```diff
@@ -0,0 +1,146 @@
+buildscript {
+    ext {
+        corda_release_group = 'net.corda'
```

Comment:
If you compare this with the other sample projects then you will see that they're all trying to extract (most of) these numbers from a shared `constants.properties` file in the root. This makes the samples easier to upgrade to the latest version of Corda etc.